code/obj/flock/structure/relay.dm

```diff
@@ -108,11 +110,12 @@
 		for (var/turf/T as anything in turfs)
 			if (istype(T, /turf/simulated) && !isfeathertile(T))
 				LAGCHECK(LAG_LOW)
-				src?.flock.claimTurf(flock_convert_turf(T))
+				src.flock?.claimTurf(flock_convert_turf(T))
```

Comment:
```suggestion
				src?.flock?.claimTurf(flock_convert_turf(T))
```

Reply:
I think if `src` is null we probably want it to runtime

Reply:
actually it might be possible, fuck it